README.md

```diff
@@ -10,4 +10,10 @@
 1. Remove method `login()` in AuthenticationService, `HashUtil` and `salt` from User entity.
    Start using `passwordEncoder` in UserService -> add() for password hashing.
 
+Advanced:
+1. In the UserRequestDto add a `repeatPassword` field.
+1. Implement validation to check that password and repeatPassword are the same.
+1. In your CustomGlobalExceptionHandler, return JSON response with HTTP status 404
```

Comment:
DataProcessingException is thrown when smt goes wrong while request processing, I don't thisnk 404 is proper status for this situation